tasks.py

```diff
@@ -9,6 +9,8 @@
 @invoke.task
 def staging(c):
     c.config.env = "staging"
+    c.config.namespace = "pressweb-staging"
```

Comment:
This probably isn't a problem since each project should override the `tasks.py` file, but maybe it's better to generalize this in the library package to something that provides an example rather than a concrete (e.g. `myproject-staging`, `myproject-web`)

This is just a nit.